docs/reference/language/scripting.md

```diff
@@ -245,6 +259,10 @@ The value in question can be either:
   [element function]($function/#element-functions) that were given when the
   element was constructed.
 
+**Important:** Many fields of element functions are only accessible in a
+concrete context that disambiguates their meaning. Consult the chapter 
```

Comment:
"disambiguates their meaning" doesn't strike me as just the right way to put it. Maybe "are only accessible in contexts where their concrete value is known."

Reply:
What I mean is: Parameter values can be different in every closure, so accessing them without specifying a closure would be ambiguous. "Where their value is known" suggests that they have no value in most situations, but actually it is the opposite: usually, there are many alternative values that would fit the query (e.g. fonts for headings, text, footnotes etc), so one has to be more specific. In my mind, set rules for different content types are all simultaneously alive, just not simultaneously active.